recipes/qbs/all/conandata.yml

```diff
@@ -0,0 +1,11 @@
+sources:
+  "1.16.0":
+    Linux:
+      url: "http://download.qt.io/official_releases/qtcreator/4.12/4.12.1/installer_source/linux_gcc_64_rhel72/qtcreator.7z"
+      md5: "ba6bfeb65c8b6a6713bfba098f7ec40c"
```

Comment:
Can you use sha256 hashes?
md5 hashes are considered insecure nowadays.

Reply:
Sure. I thought this is only about integrity check and only the md5sums were published on the Qt download page.